### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/deployment/https.md`

### PR DESCRIPTION
**Pull request by xzmeng**

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit e69c5c1090fc79acd6c2f66abd2947210752f9f5 at: https://759ff0cb.fastapitiangolo.pages.dev

File: docs/zh/docs/deployment/https.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
+# 关于 HTTPS
+
+人们很容易认为 HTTPS 仅仅是“启用”或“未启用”的东西。
+
+但实际情况比这复杂得多。
+
+!!!提示
+     如果您很赶时间或不在乎，请继续阅读下一部分，下一部分会提供一个step-by-step的教程，告诉你怎么使用不同技术来把一切都配置好。
+
+要从顾客的角度**了解 HTTPS 的基础知识**，请查看 <a href="https://howhttps.works/" class="external-link" target="_blank">https://howhttps.works/</a>。
```
Comment (shamosishen) [inline docs/zh/docs/deployment/https.md:10]:
```suggestion
要从用户的角度**了解 HTTPS 的基础知识**，请查看 <a href="https://howhttps.works/" class="external-link" target="_blank">https://howhttps.works/</a>。
```

File: docs/zh/docs/deployment/https.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+现在，从**开发人员的角度**，在考虑 HTTPS 时需要记住以下几点：
+
+* 对于 HTTPS，**服务器**需要拥有由**第三方**生成的**"证书(certificate)"**。
+     * 这些证书实际上是从第三方**获取**的，而不是“生成”的。
+* 证书有**生命周期**。
+     * 它们会**过期**。
+     * 然后它们需要**更新**，**再次从第三方获取**。
+* 连接加密发生在 **TCP 层**。
```
Comment (shamosishen) [inline docs/zh/docs/deployment/https.md:19]:
```suggestion
* 连接的加密发生在 **TCP 层**。
```

File: docs/zh/docs/deployment/https.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+     * 这是 HTTP **下面的一层**。
+     * 因此，**证书和加密**处理是在 **HTTP之前**完成的。
+* **TCP 不知道“域(domains)”**。 仅仅知道 IP 地址。
```
Comment (shamosishen) [inline docs/zh/docs/deployment/https.md:22]:
```suggestion
* **TCP 不知道“域(domain)”**。 仅仅知道 IP 地址。
```

File: docs/zh/docs/deployment/https.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+     * 有关所请求的 **特定域(specific domain)** 的信息位于 **HTTP 数据**中。
+* **HTTPS 证书**“证明”**某个域**，但协议和加密发生在 TCP 层，在知道正在处理哪个域**之前**。
+* **默认情况下**，这意味着您**每个 IP 地址只能拥有一个 HTTPS 证书**。
+     * 无论您的服务器有多大，或者服务器上的每个应用程序有多小。
+     * 不过，对此有一个**解决方案**。
+* **TLS** 协议（在 HTTP 之下的TCP 层处理加密的协议）有一个**扩展**，称为 **<a href="https://en.wikipedia.org/wiki/ Server_Name_Induction" class="external-link" target="_blank"><abbr title="服务器名称指示">SNI</abbr></a>**。
```
Comment (shamosishen) [inline docs/zh/docs/deployment/https.md:28]:
```suggestion
* **TLS** 协议（在 HTTP 之下的TCP 层处理加密的协议）有一个**扩展**，称为 **<a href="https://en.wikipedia.org/wiki/ Server_Name_Indication" class="external-link" target="_blank"><abbr title="服务器名称指示">SNI</abbr></a>**。
```

File: docs/zh/docs/deployment/https.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+* **获得安全连接后**，通信协议**仍然是HTTP**。
+     * 内容是 **加密过的**，即使它们是通过 **HTTP 协议** 发送的。
+
+通常的做法是在服务器上运行**一个程序/HTTP 服务器**并**管理所有 HTTPS 部分**：接收**加密的 HTTPS 请求**， 将 **解密的 HTTP 请求** 发送到在同一服务器中运行的实际 HTTP 应用程序（在本例中为 **FastAPI** 应用程序），从应用程序中获取 **HTTP 响应**，**对其进行加密** 使用适当的 **HTTPS 证书** 并使用 **HTTPS** 将其发送回客户端。 此服务器通常称为 **<a href="https://en.wikipedia.org/wiki/TLS_termination_proxy" class="external-link" target="_blank">TLS 终止代理(TLS Termination Proxy)</a>**。
+
+您可以用作 TLS 终止代理的一些选项包括：
+
+* Traefik（也可以处理证书更新）
+* Caddy（也可以处理证书更新）
+* nginx
```
Comment (shamosishen) [inline docs/zh/docs/deployment/https.md:40]:
```suggestion
* Nginx
```

**Review by shamosishen — CHANGES_REQUESTED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit faca68b3bce85df63a1a6987a2bf9c843fc6479b at: https://ef1bebd9.fastapitiangolo.pages.dev

File: docs/zh/docs/deployment/https.md [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+
+### 域名
+
+这可能都是从您**获取**一些**域名**开始的。 然后，您可以在 DNS 服务器（可能是您的同一家云服务商）中配置它。
+
+您可能会获得一个云服务器（虚拟机）或类似的东西，并且它会有一个<abbr title="That isn't Change">固定</abbr> **公共IP地址**。
+
+在 DNS 服务器中，您可以配置一条记录（“A 记录”）以将 **您的域** 指向您服务器的公共 **IP 地址**。
+
+在设置所有内容时，您可能只会在第一次时执行此操作一次。
```
Comment (socket-socket) [inline docs/zh/docs/deployment/https.md:69]: change

File: docs/zh/docs/deployment/https.md [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+
+!!! tip
+     域名早于 HTTPS，但由于一切都依赖于域和 IP 地址，因此值得在此提及。
```
Comment (socket-socket) [inline docs/zh/docs/deployment/https.md:72]: change

**Review by socket-socket — CHANGES_REQUESTED** [review]

**Comment by socket-socket** [comment]
”域“和”域名“是否是相同的？

**Comment by tiangolo** [comment]
📝 Docs preview for commit 14dd7ca9f68d9d71427054fba54dc0b298d4aff8 at: https://34e98a81.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit a60bb0411bbb0160cd1dda9a6032282729ec1955 at: https://11667024.fastapitiangolo.pages.dev

File: docs/zh/docs/deployment/https.md [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+     * 这是 HTTP 协议**下面的一层**。
+     * 因此，**证书和加密**处理是在 **HTTP之前**完成的。
+* **TCP 不知道“域(domain)”**。 仅仅知道 IP 地址。
+     * 有关所请求的 **特定域(specific domain)** 的信息位于 **HTTP 数据**中。
+* **HTTPS 证书**“证明”**某个域**，但协议和加密发生在 TCP 层，在知道正在处理哪个域**之前**。
+* **默认情况下**，这意味着你**每个 IP 地址只能拥有一个 HTTPS 证书**。
+     * 无论你的服务器有多大，或者服务器上的每个应用程序有多小。
+     * 不过，对此有一个**解决方案**。
+* **TLS** 协议（在 HTTP 之下的TCP 层处理加密的协议）有一个**扩展**，称为 **<a href="https://en.wikipedia.org/wiki/Server_Name_Indication" class="external-link" target="_blank"><abbr title="服务器名称指示">SNI</abbr></a>**。
+     * SNI 扩展允许一台服务器（具有 **单个 IP 地址**）拥有 **多个 HTTPS 证书** 并提供 **多个 HTTPS 域/应用程序**。
```
Comment (socket-socket) [inline docs/zh/docs/deployment/https.md:29]:
```suggestion
     * SNI 扩展允许一台服务器（具有 **单个 IP 地址**）拥有 **多个 HTTPS 证书** 并提供 **多个 HTTPS 域名/应用程序**。
```

File: docs/zh/docs/deployment/https.md [hunk 9]
```diff
@@ -0,0 +1,192 @@
...
+
+在未来的某个时候，每个证书都会**过期**（大约在获得证书后 3 个月）。
+
+然后，会有另一个程序（在某些情况下是另一个程序，在某些情况下可能是同一个 TLS 终止代理）与 Let's Encrypt 通信并更新证书。
+
+<img src="/img/deployment/https/https.svg">
+
+**TLS 证书** **与域名相关联**，而不是与 IP 地址相关联。
+
+因此，要更新证书，更新程序需要向权威机构（Let's Encrypt）**证明**它确实**“拥有”并控制该域**。
```
Comment (socket-socket) [inline docs/zh/docs/deployment/https.md:172]:
```suggestion
因此，要更新证书，更新程序需要向权威机构（Let's Encrypt）**证明**它确实**“拥有”并控制该域名**。
```

File: docs/zh/docs/deployment/https.md [hunk 10]
```diff
@@ -0,0 +1,192 @@
...
+
+现在客户端和服务器（特别是浏览器和 TLS 终止代理）具有 **加密的 TCP 连接**，它们可以开始 **HTTP 通信**。
+
+接下来，客户端发送一个 **HTTPS 请求**。 这其实只是一个通过 TLS 加密连接的 HTTP 请求。
+
+<img src="/img/deployment/https/https04.svg">
+
+### 解密请求
+
+TLS 终止代理将使用协商好的加密算法**解密请求**，并将**（解密的）HTTP 请求**传输到运行应用程序的进程（例如运行 FastAPI 应用的 Uvicorn 进程） 。
```
Comment (socket-socket) [inline docs/zh/docs/deployment/https.md:132]:
```suggestion
TLS 终止代理将使用协商好的加密算法**解密请求**，并将**（解密的）HTTP 请求**传输到运行应用程序的进程（例如运行 FastAPI 应用的 Uvicorn 进程）。
```

**Review by socket-socket — APPROVED** [review]

**Review by shamosishen — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 9c581bb08adf64b522d0d5e11f5b0cedc34d7b04 at: https://e06746b4.fastapitiangolo.pages.dev

**Review by Aruelius — APPROVED** [review]

**Comment by tiangolo** [comment]
Thank you @xzmeng! 🍰 

And thanks for the reviews! @shamosishen, @Aruelius, @socket-socket 🙇

**Pull request merged**